### DISCUSSION TITLE
Do you see my PR (JS/Canvas)

### DISCUSSION THREAD

**Discussion by nevrozza** [chrisbanes/haze#231] [Q&A] [answered]
Hi! I'm new to GitHub so idk if you see my PR about JS/Canvas. Previous build failed, but I have updated my repo. It should work now! 
https://github.com/chrisbanes/haze/pull/228

**Comment by chrisbanes** [comment, marked as answer]
Hi. Yes I'm on vacation at the moment, so will look when I'm back 🙂
